opus/import/obs_cassini_common.py

```
@@ -198,6 +389,65 @@ def _cassini_normalize_mission_phase_name(self):
                 return phase.upper()
         return None
 
+    #################################################################
+    ### HELPER FUNCTIONS USED BY METHODS FOR obs_instrument_coiss ###
+    #################################################################
+    # See additional notes under _COISS_FILTER_WAVELENGTHS
+    def _coiss_wavelength_helper(self, camera, filter1, filter2):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_coiss_wavelength_helper`

(ANN202)